db/index.js

```diff
@@ -10,17 +10,36 @@ let writeFile = util.promisify(fs.writeFile);
 const filePath = path.resolve('db/cameras.json');
 
 async function readCameras() {
-  const json = await readFile(filePath);
+  const json = await readFile(filePath, 'utf8');
   return JSON.parse(json);
 }
 
 async function writeCameras(cameras) {
   const json = JSON.stringify(cameras, null, 2);
   await writeFile(filePath, json);
+
+  const json = JSON.stringify(items, null, 2);
+  return writeFile(dbfilepath, json, 'utf8')
 }
 
 async function createCamera(country) {
   const allCountries = await readCountries();
```

Comment:
const allCountries = await readCountries(); 

You might to change that from countries to Cameras

Reply:
Thats strange. I checked my code and I dont have allCountries. It is instead allCameras.